executor/executor_test.go

```diff
@@ -5609,6 +5609,13 @@ func TestAdmin(t *testing.T) {
 	result.Check(testkit.Rows(fmt.Sprintf("%d %s", historyJobs[3].ID, historyJobs[3].Query), fmt.Sprintf("%d %s", historyJobs[4].ID, historyJobs[4].Query)))
 	result = tk.MustQuery(`admin show ddl job queries limit 3 offset 2`)
 	result.Check(testkit.Rows(fmt.Sprintf("%d %s", historyJobs[2].ID, historyJobs[2].Query), fmt.Sprintf("%d %s", historyJobs[3].ID, historyJobs[3].Query), fmt.Sprintf("%d %s", historyJobs[4].ID, historyJobs[4].Query)))
+	// check situations when `admin show ddl job queries limit 3 offset 2` happens at the same time with new DDLs being executed
+	go func() {
```

Comment:
this goroutine most probably runs before next DDL queries in line 5617~5618, because if line 5617~5618 runs, the ddl query list will have two new records at the front so `admin show ddl job queries limit 3 offset 2` will see the 0~2nd of `historyJobs`. Now the check of 2~4th is passed which is a wrong test

you can start two goroutine, one for repeatly drop+create table, the other one for checking the result set of `admin show ddl job queries limit 3 offset 2` has no duplication. Don't forget to use sync.WaitGroup to wait them exit